<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndQrip</class>
 <widget class="QMainWindow" name="wndQrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q Rip - The CD Ripper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pix/ic_newdvd.png</normaloff>:/pix/ic_newdvd.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>CDROM Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDevice">
        <property name="text">
         <string>/dev/cdrom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGo">
        <property name="text">
         <string>&amp;Go !</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMultiple">
        <property name="text">
         <string>Multiple works CD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lblDestData_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Album Title (%T)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAlbumTitle">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lblDestData_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Album Artist (%A)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAlbumArtist">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lblDestData_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Album Genre (%G)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAlbumGenre">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="twTracks">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Extract</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Composer (%C)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Work (%W)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track no (%N)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Track title (%P)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Length (%L)</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblDestData">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Data destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDestData">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDestData_5">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Output format (%E)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFormat">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>flac</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>wav</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mp3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ogg</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbPlaylist">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Play list destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leDestPlaylist">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDestData_2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Output format (%M)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFormatPlaylist">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>m3u</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QToolButton" name="tbEject">
        <property name="text">
         <string>Eject !</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbEject">
        <property name="text">
         <string>Eject when done</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbRemove">
        <property name="text">
         <string>Remove temp files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOverwrite">
        <property name="text">
         <string>Overwrite files</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbLive">
        <property name="text">
         <string>Live tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="tbExit">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="qrip.qrc">
          <normaloff>:/pix/ic_exit.png</normaloff>:/pix/ic_exit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblProgress">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbExtract">
        <property name="text">
         <string>Extract</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="qrip.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbExit</sender>
   <signal>clicked()</signal>
   <receiver>wndQrip</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
